implementations/rust/ockam/ockam_examples/example_projects/tunnel/src/bin/client.rs

```diff
@@ -0,0 +1,36 @@
+use ockam::{
+    route, Context, Entity, Result, SecureChannels, TcpTransport, TrustEveryonePolicy, Vault, TCP,
+};
+
+const OUTLET_NAME: &str = "outlet";
+
+#[ockam::node]
+async fn main(mut ctx: Context) -> Result<()> {
+    // get the port argument
+    let port = std::env::args()
+        .nth(1)
+        .expect("provide the local port as the first argument to the binary");
+
+    // create secure listener
+    let vault = Vault::create(&ctx)?;
+    let mut me = Entity::create(&ctx, &vault)?;
+    me.create_secure_channel_listener("secure_listener", TrustEveryonePolicy)?;
```

Comment:
Is there a purpose for this listener?

Reply:
Whoops, that was a leftover from code before!